packages/backend/src/services/DashboardService/DashboardService.ts

```diff
@@ -324,7 +325,11 @@ export class DashboardService {
 
         const duplicatedDashboard = {
             ...dashboard,
-            name: `Copy of ${dashboard.name}`,
+            description: data.dashboardDesc,
+            name:
+                dashboard.name === data.dashboardName
+                    ? `Copy - ${dashboard.name}`
+                    : data.dashboardName,
```

Comment:
Since we are not constraining dashboard/chart names to be unique within the organization, I don't see a reason why we should maintain this check in the backend. I think frontend change is enough!